lua/yabs/init.lua

```diff
@@ -224,15 +255,21 @@ function Yabs:run_default_task()
     end
 end
 
-local function file_exists(file)
-    local f = io.open(file, "rb")
-    if f then f:close() end
-    return f ~= nil
-end
-
 function Yabs:load_config_file()
     if file_exists(".yabs") then
-        dofile(vim.fn.getcwd() .. "/.yabs")
+        local config = dofile(vim.fn.getcwd() .. "/.yabs")
```

Comment:
```suggestion
        local config = dofile(vim.loop.cwd() .. "/.yabs")
```

But I would consider using `Plenary.Path` for path concatenation.